### PR TITLE
Restore offline playability with this one weird secret!

### PR DESCRIPTION
**Pull request by chadwhitacre**
I am on a road trip for six weeks. Sometime I do not have Internet access, but wish to play A Dark Room nonetheless. I believe this change will allow me to do so. ☺️ 🙇

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -21,6 +21,7 @@
 	<script>
 		if(!window.jQuery) {
```
Comment (whit537) [inline index.html]: Assume that if we can't find jQuery, it means we're offline and also can't find the Dropbox script.

**Comment by groteworld** [comment]
I approve of this clickbait!

So there is no configuration, just adding the library adds some offline functionality? Wouldn't app caching with a manifest work the same? and if so, what benefit do we get from this (128kb file) over a manifest file?

Thanks for your continued support of ADR, I know I've been thinking of your issues, and apologize for the radio silence.

**Comment by Continuities** [comment]
I don't even think the Dropbox stuff is currently working due to the lack of https. Setting that up has been on my list forever...

**Comment by groteworld** [comment]
@whit537 will have to confirm, but i don't think this actually uses dropbox like we used it for save files, I believe there is initialization code in this library that sets up some offline usage if disconnected. If that is the case, the need for https is unnecessary, though I wish we could!

**Comment by whit537** [comment]
> I approve of this clickbait!

😁 

> I don't even think the Dropbox stuff is currently working
> i don't think this actually uses dropbox like we used it for save files

To be honest, I didn't delve into understanding what ADR does with `dropbox.js`. My goal was to make the smallest change possible to work around the `Dropbox is not defined` error when loading the game without an Internet connection. It seemed simplest to do the same for the Dropbox script as for the jQuery script.

# Before

screen shot 2017-07-29 at 11 16 11 pm

# After

screen shot 2017-07-29 at 11 16 34 pm

**Comment by whit537** [comment]
But to be fair, retesting now, the game appears to be playable even with the `Dropbox is not defined` error present. So feel free to close this if reviewing this PR is more trouble than it's worth. ☺️

**Comment by whit537** [comment]
(... and, I'm also done with the roadtrip! 😬 )

**Pull request closed without merging**